### ISSUE TITLE
v2.7.3 seems to have broken the rename downloads feature

### ISSUE THREAD

**Issue by FatOrangutan** [gyng/save-in#80]
STR:
1. Add the following rule:
```
filename: (.*)(:|-|=)(large|small|medium|thumb|orig)
sourceurl: pbs.twimg.com
capture: filename
into: :$1:
```
2. Go to https://pbs.twimg.com/media/Di6uEBuVsAEYVBw.jpg:orig
3. Right click > Save In...

ER / AR with 2.7.1: download is renamed to `Di6uEBuVsAEYVBw.jpg`. Screesnhot:
![2 7 1](https://user-images.githubusercontent.com/8336638/43230792-51782276-9087-11e8-873e-49ff558cdf78.PNG)

AR with 2.7.3: Save As download window shows up with prepopulated name as `Di6uEBuVsAEYVBw.jpg_orig`. Screenshot:
![2 7 3](https://user-images.githubusercontent.com/8336638/43230824-75af8a4e-9087-11e8-824e-cba2da37a526.PNG)

**Comment by gyng** [comment]
Released 3.0.0 with a fix for this. I've done some testing on Chrome and Firefox, so it *should* be OK.

**Comment by gyng** [comment]
I'm assuming this is fixed, please reopen if that's not the case.
